src/shared-components/alert/test.tsx

```diff
@@ -1,65 +1,62 @@
-import React from 'react';
+import React, { ReactElement } from 'react';
 import TestRenderer from 'react-test-renderer';
-import { shallow } from 'enzyme';
 
 import Alert from './index';
 
-const testAlert = {
-  text: 'Your email address was updated successfully!',
-  type: 'success',
-  duration: 'sticky',
-};
+// Note on truncateText prop test: this cannot be tested because element scrollHeight and offsetHeight are not simulated correctly
+
+const alertText =
+  'Lorem ipsum, dolor sit amet consectetur adipisicing elit. Ex quis dicta iusto a placeat!';
```

Comment:
Are we using this because it's longer than `Your email address was updated successfully!`?

Reply:
yes that was the idea, 80% of the time on this PR was trying to make a truncateText test, but found out that those dom props like scrollHeight that we use in the Component are all hardcoded to 0 in tests so desisted (we probably can make it work but not worth the effort since we have a real test example in story anyway)

I can change this lorem text to `Your email address was updated successfully!` now